ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountReportsDao.java

```diff
@@ -80,6 +85,9 @@
    */
   void updateStorageUsage(String clusterName, String hostname, int partitionId, short accountId, short containerId,
       long storageUsage) throws SQLException {
+    // TODO: adding real physical storage usage and number of blobs here
+    long physicalStorageUsage = storageUsage;
+    long numberOfBlobs = 1L;
     try (Connection connection = dataSource.getConnection()) {
       try (PreparedStatement insertStatement = connection.prepareStatement(insertSql)) {
```

Comment:
Do we need to escape the sql statement to guard ourselves against the sql injections? This applies to all other sql statements in this file.

Reply:
I thought prepared statement would differentiate the sql code from the user input, thus prevent sql injections.